test/unit/domain/shared/amount-calculator.spec.ts

```diff
@@ -150,4 +150,15 @@ describe("AmountCalculator", () => {
       })
     }
   })
+
+  describe("multiplies", () => {
+    it("correctly applies basis points to an amount", () => {
```

Comment:
please add a test where the amount should be rounded and other which return min value (with a small amount)